tdm-be/src/worker/enrichmentHook.ts

```diff
@@ -75,17 +75,17 @@ const enrichmentHookSuccess = async (processingId: string) => {
         return enrichmentUrl.includes(entry.url);
     });
 
-    // Check if enrichment entry existe
+    // Check if enrichment entry exist
     if (!enrichmentEntry || !enrichmentEntry.url || !enrichmentEntry.retrieveUrl) {
-        error(processingId, 'Enrichment-Hook config dos not contait the enrichment url');
+        error(processingId, 'Enrichment-Hook config dos not contain the enrichment url');
         // Send error the global catcher because this is normally impossible
-        throw new Error('This is normally impossible - Enrichment-Hook config dos not contait the enrichment url');
+        throw new Error('This is normally impossible - Enrichment-Hook config dos not contain the enrichment url');
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
        throw new Error('This is normally impossible - Enrichment-Hook config does not contain the enrichment url');
```